system-addon/bootstrap.js

```diff
@@ -30,8 +30,16 @@ let waitingForBrowserReady = true;
 XPCOMUtils.defineLazyModuleGetter(this, "ActivityStream",
   "resource://activity-stream/lib/ActivityStream.jsm", null, null, () => {
     // Helper to fetch a resource directory listing and call back with each item
-    const processListing = async (uri, cb) => (await (await fetch(uri)).text())
-      .split("\n").slice(2).forEach(line => cb(line.split(" ").slice(1)));
+    const processListing = async (uri, cb) => {
+      try {
+        (await (await fetch(uri)).text())
+          .split("\n").slice(2).forEach(line => cb(line.split(" ").slice(1)));
+      } catch (e) {
+        // Silently ignore any modules that fail to load.
```

Comment:
The modules aren't being loaded here, so this comment is confusing. Maybe just "Silently ignore listings that fail to load -- probably because the resource: has been unloaded"